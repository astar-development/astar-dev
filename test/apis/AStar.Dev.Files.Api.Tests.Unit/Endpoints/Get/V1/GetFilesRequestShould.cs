using AStar.Dev.Utilities;

namespace AStar.Dev.Files.Api.Endpoints.Get.V1;

public class GetFilesRequestShould
{
    [Fact]
    public void ContainTheExpectedProperties()
        => new GetFilesRequest().ToJson().ShouldMatchApproved();
}
