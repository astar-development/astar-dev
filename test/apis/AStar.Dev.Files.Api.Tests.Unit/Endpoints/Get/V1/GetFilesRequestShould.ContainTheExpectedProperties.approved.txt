{
  "searchFolder": "",
  "recursive": true,
  "excludeViewedWithinDays": 0,
  "includeMarkedForDeletion": false,
  "searchText": null,
  "currentPage": 1,
  "itemsPerPage": 10,
  "sortOrder": "SizeDescending",
  "searchType": "Duplicates",
  "name": "Get Files",
  "httpMethod": "GET"
}